src/pretalx/person/signals.py

```diff
@@ -1,6 +1,28 @@
 from django.dispatch import receiver
 
-from pretalx.common.signals import register_data_exporters
+from pretalx.common.signals import EventPluginSignal, register_data_exporters
+
+html_above_person_form = EventPluginSignal()
+"""
+This signal is sent out to display additional information on the speaker
+pages in the internal organiser area.
+
+As with all plugin signals, the ``sender`` keyword argument will contain the
+event. Additionally, the signal will be called with the ``request`` it is
+processing, and the ``person`` which is currently displayed.
+The receivers are expected to return HTML.
+"""
```

Comment:
I’m not sure I like this – this way we’d accumulate a lot of these signals. I think it’s worth taking a look at what [pretix](https://docs.pretix.eu/en/latest/development/api/general.html#id1) is doing in instances like this – pretix allows plugins to add more forms to given pages, or to even replace the form class of a certain page, see e.g. `pretix.control.signals.item_forms`, `pretix.control.signals.voucher_form_class`.

In any case, I’d probably restrict this to a `pretalx.orga.speaker_form_html` signal that is rendered below the form, and if there is a super urgent need for the plugin to add content to the top of the page, it could still move content up via JS, I suppose.

Reply:
Thanks a lot for pointing that out! That approach is definitely more elegant. I'll dig into that.
The reason for me to put one hook above and one below here was that we actually have the usecase that we have some information that belongs below the form. Information from other systems like RT in particular but also Zammad and Frab. They are obviously all read-only now, btw. which could be adressed with the approach you suggested.
But we will also have some information that will be 'urgent' and should be on top of the page: during the congress, when a speaker has not shown up short before the presentation, that is the kind of indication I'd like to see above anything else.

Reply:
I still don't really like the idea to reorder the page using JS but removed the 'above_xxx' hooks.

Reply:
> ... I’d probably restrict this to a `pretalx.orga.speaker_form_html` signal ...

Personally, I'd prefer to have the signals in the namespace of the ,odels they are related to. But the couly of courde all be moved into the  `pretalx.orga` space if you like that more!